sw/device/silicon_creator/rom/data/rom_manual_testplan.hjson

```diff
@@ -0,0 +1,29 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+{
+  name: "rom_manual"
```

Comment:
I am not sure if there is a top level `desc` field for test plans, but I think we should add a description of what this test plan is trying to accomplish (e.g. sign-off criteria).

Reply:
IMHO all testplans have the same goal, right? I thought this was clear enough but we should probably talk about this in the sw sync (or maybe a wider forum).

@timothytrippel does a `desc` field exist or @moidx would it be enough to add a comment in the hjson?

Reply:
I am also thinking about people not familiar with the project, so that it is easier for them to tie all the pieces together.

Reply:
There is no top level `desc` field for testplans that I am aware of, however, you could:
1. add one, since the current DV tooling doesn't read such a field anyway; moreover this is not ingested by any DV tooling, or
2. add a comment with the top level description.